mongodump/mongodump_test.go

```diff
@@ -832,3 +832,44 @@ func TestMongoDumpOplog(t *testing.T) {
 
 	})
 }
+
+// Test dumping a capped collection with autoIndexId:false.  As of MongoDB 4.0,
```

Comment:
are we testing with capped for a reason other than that this was the user example? i.e. did this problem exist for non-capped collections with autoIndexId:false, too? just wondering sure if there's some relationship b/w capped collections and autoIndexId that I'm not aware of.

Reply:
I tested capped because that was the case, but I spot checked and it applies to non-capped, too.  I'll drop back to non-capped for testing for clarity.